app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -78,6 +78,15 @@ class LoginFragment : Fragment() {
             loginActivityViewModel.sendResetPasswordEmail(email.text.toString())
         }
 
+        loginActivityViewModel.user.observe(this, Observer {
+            redirectToMain()
+        })
+
+        loginActivityViewModel.loggedIn.observe(this, Observer {
+            Toast.makeText(context, "Success!", Toast.LENGTH_LONG).show()
+            loginActivityViewModel.fetchProfile()
+        })
```

Comment:
` loginActivityViewModel.loggedIn.observe(this, Observer {
`
is already there don't add it again
add the above two lines there

Reply:
My bad! Will resolve it :+1:

Reply:
PR Updated